policyengine/utils/huggingface.py

```diff
@@ -4,7 +4,10 @@
 
 
 def download(
-    repo: str, repo_filename: str, local_folder: str, version: str = None
+    repo: str,
+    repo_filename: str,
+    local_folder: str | None = None,
+    version: str | None = None,
 ):
     token = os.environ.get("HUGGING_FACE_TOKEN")
```

Comment:
__suggestion:__ Modify to not require token for public datasets

Akin to my open PR in `-core`, I think we'll want to (potentially later) modify this so that a token isn't required for public datasets, e.g., `-us`.

Reply:
Yes- filed #64